modules/m_services.c

```diff
@@ -139,6 +139,11 @@ me_su(struct MsgBuf *msgbuf_p, struct Client *client_p, struct Client *source_p,
 	sendto_common_channels_local(target_p, CLICAP_ACCOUNT_NOTIFY, NOCAPS, ":%s!%s@%s ACCOUNT %s",
```

Comment:
Exclude extended-monitor

```suggestion
	sendto_common_channels_local(target_p, CLICAP_ACCOUNT_NOTIFY, CLICAP_EXTENDED_MONITOR, ":%s!%s@%s ACCOUNT %s",
```